src/buildCommon.js

```diff
@@ -183,6 +184,11 @@ const mathit = function(
             fontName: "Main-Italic",
             fontClass: "mainit",
         };
+    } else if (value.charCodeAt(0) === 0xD835) {
+        return {
+            fontName: "Math-Italic",
```

Comment:
Are metrics the same as `Math-Italic`?

Reply:
> Are missing characters intentional?

The missing sans-serif "a" is not intentional. I don't see any other missing characters. This is a PR for characters in the Unicode range from U+1D4D0 to U+1D4E9. A letter like "ℂ" does not not belong in this PR because its code point is U+2102, in the Unicode basic multilingual plane.

> Are metrics identical with Math-Italic?

No, but they are much closer to correct than the just using the metrics for the letter "M", as we do elsewhere for some Unicode characters. The biggest discrepancy is in the Fraktur lower case letters. Some of them, such as 𝔷, have descenders that are not present in Math-Italic.

Reply:
I'm a little confused why you'd need to be hacking metrics like this.  Is this PR using fallback fonts?  I think that's something that KaTeX generally tries to avoid, instead using characters from its own fonts, and adding to the fonts if needed.  Could you clarify which characters in this PR are supported by KaTeX fonts vs. being hacked in?

Reply:
Error on my part. I wasn't aware that we had font metrics for fonts other than Main and AMS. Now, having actually looked into file `fontMetricsData`, I see that I don't need to use a proxy for the metrics.

I'll revise the PR to use the correct metrics. No proxy is necessary. This PR calls only on KaTeX fonts and does not have a system font fall back.